draft-ietf-quic-transport.md

```diff
@@ -3981,6 +3979,12 @@ A client MUST NOT include an original connection ID, a stateless reset token, or
 a preferred address.  A server MUST treat receipt of any of these transport
 parameters as a connection error of type TRANSPORT_PARAMETER_ERROR.
 
+max_connection_ids (0x000e):
+
+: The maximum number of connection IDs that the peer is willing to store.
+  This value includes only connection IDs sent in NEW_CONNECTION_ID frames.
+  If this parameter is absent, a default of 8 is assumed.
```

Comment:
I'd prefer a default of 0.

Reply:
I'm not sure I see a default of 0 as being better than 8. 
If we're going to introduce a TP, and if everyone is going to set the TP all the time, then the default doesn't matter. If not, then I think we want to maintain 8 (or some similar value) for the same reasons that we had it before there was a TP.

Reply:
The point is that otherwise everyone is going to change it from 8. Using 0 avoids the discussion of who's default is better and is in line with other 0 defaults such as max streams.

Reply:
Agreed, we've consistently gone with 0 being a default whenever possible, even in cases when 0 makes the transport basically useless.

Reply:
Then that works for me!

Reply:
Done.

Reply:
I we have default = 0, do we also need a "migration not supported" TP? It would seem that not setting a non null value would effectively prevent migration.

Reply:
It would, but you can support CID rotation *without* supporting migration, so the reverse isn't true.